### PR TITLE
sbmlexport: user can optionally specify output file

### PR DESCRIPTION
**Pull request by freestraws**

### CODE REVIEW

**Comment by freestraws** [comment]
i used http://effbot.org/zone/element-lib.htm#prettyprint for the indentation/ pretty-formatted output.

**Comment by jonls** [comment]
@freestraws Did you try `nargs='?'` as mentioned above?

File: psamm/commands/sbmlexport.py [hunk 1]
```diff
@@ -24,6 +24,15 @@
 class SBMLExport(MetabolicMixin, Command):
     """Export model as SBML file."""
 
+    @classmethod
+    def init_parser(cls, parser):
+        parser.add_argument(
+            'file', type=str, help='File path for writing the Excel workbook',
+            default='-')
```
Comment (jonls) [inline psamm/commands/sbmlexport.py]: Should call the super method as the last thing in this method. See other commands for example.

File: psamm/datasource/sbml.py [hunk 2]
```diff
@@ -1020,4 +1035,5 @@ def write_model(self, file, model):
         self._add_gene_list(model_tag, gene_ids)
 
         tree = ET.ElementTree(root)
+        self._indent(root)
```
Comment (jonls) [inline psamm/datasource/sbml.py]: I would like to only apply this if an option `--pretty` is provided to the `sbmlexport` command.

File: docs/commands.rst [hunk 3]
```diff
@@ -388,9 +388,17 @@ SBML Export (``sbmlexport``)
 
 Exports the model to the SBML file format.
 
+The below will export the model to the terminal
+
+.. code-block:: shell
+
+    $ psamm-model sbmlexport -
+
+Using `--dest` option allows saving this output to a file:
```
Comment (jonls) [inline docs/commands.rst]: This was changed to positional argument.

File: psamm/commands/sbmlexport.py (hunk 1 above)
Comment (freestraws) [inline psamm/commands/sbmlexport.py]: Ooooh, I was looking at `excelexport` for reference and it doesn't use `nargs` or its `init_parser` for some reason, I thought that was the way to go.

File: psamm/commands/sbmlexport.py [hunk 4]
```diff
@@ -24,6 +24,19 @@
 class SBMLExport(MetabolicMixin, Command):
     """Export model as SBML file."""
 
+    @classmethod
+    def init_parser(cls, parser):
+        parser.add_argument(
+            'file', type=str, help='File path for writing the Excel workbook',
```
Comment (jonls) [inline psamm/commands/sbmlexport.py]: 'Excel workbook' -> 'SBML model'

**Comment by jonls** [comment]
Merged with minor changes in #195

**Pull request closed without merging**